sdk/ml/azure-ai-ml/azure/ai/ml/operations/_datastore_operations.py

```diff
@@ -43,11 +48,13 @@ def __init__(
         operation_scope: OperationScope,
         operation_config: OperationConfig,
         serviceclient_2023_04_01_preview: ServiceClient042023Preview,
```

Comment:
Is the 2023_04_01_preview client still needed, or could all operations be replaced with the new client handle?

Reply:
Do you know who (member of which team) and when (on what occasions) updates service client version? It may be risky since there may be breaking changes on service side for different api-version. I'm not sure if all operations can be replaced with the new client handle.